proto/sentry_protos/snuba/v1/endpoint_get_traces.proto

```diff
@@ -0,0 +1,52 @@
+syntax = "proto3";
+
+package sentry_protos.snuba.v1;
+
+import "sentry_protos/snuba/v1/request_common.proto";
+import "sentry_protos/snuba/v1/trace_item_attribute.proto";
+import "sentry_protos/snuba/v1/trace_item_filter.proto";
+
+message TraceColumn {
+  enum Name {
+    NAME_UNSPECIFIED = 0;
+    NAME_TRACE_ID = 1;
+    NAME_START_TIMESTAMP = 2;
+    NAME_ROOT_SPAN_NAME = 3;
+    NAME_TOTAL_SPAN_COUNT = 4;
+    NAME_FILTERED_SPAN_COUNT = 5;
+  }
+
+  Name name = 1;
+  AttributeKey.Type type = 2;
+}
+
+message GetTracesRequest {
+  message OrderBy {
+    TraceColumn column = 1;
+    bool descending = 2;
+  }
+
+  RequestMeta meta = 1;
+  TraceItemFilter filter = 2;
```

Comment:
this won't work for searching across event types. At least not by itself. The filter currently does not support a `name`. That's why in the other file there's extra filter definitions for traces (because they are different).

Reply:
I didn't want to enable filtering on traces, I want to only allow filtering on trace items, that's why I removed it.

Traces don't have a name, they don't have attributes, I don't think we should enable filtering on things that are defined by trace item fields.

Reply:
with your current API, how would you find all traces where `span.op = http` and `error.project_id=1`?

Reply:
Pass a `TraceItemFilter` with your conditions, we map the prefixes to each dataset and we query.

Reply:
or event better, show me an example with python

Reply:
I think this should fix your concerns: https://github.com/getsentry/sentry-protos/pull/71/commits/ce87464fd6b10afabc9da96031963c2ea52ffdff .